tools/pipeline_perf_test/test_suites/integration/configs/integration_report_logs_filtered.yaml

```diff
@@ -0,0 +1,323 @@
+queries:
+  - name: Filter superflous metadata
+    sql: |
+      CREATE VIEW metadata_filtered AS
+      SELECT *
+      FROM metadata
+      WHERE Attribute NOT LIKE 'test.ctx.%';
+
+  - name: Calculate observation window for this test
```

Comment:
I'm not sure how to review this file. Should it have a README? I wonder if there's something re-usable in here, or a pattern I haven't spotted yet.